lib/node_modules/@stdlib/stats/incr/nanvariance/lib/index.js

```diff
@@ -0,0 +1,74 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+/**
+* Compute an unbiased sample variance incrementally, ignoring NaN values.
+*
+* @module @stdlib/stats/incr/nanvariance
+*
+* @example
+* var incrnanvariance = require( '@stdlib/stats/incr/nanvariance' );
+*
+* var accumulator = incrnanvariance();
+*
+* var v = accumulator();
+* // returns null
+*
+* v = accumulator( 2.0 );
+* // returns 0.0
+*
+* v = accumulator( NaN );
+* // returns 0.0
+*
+* v = accumulator( -5.0 );
+* // returns 24.5
+*
+* v = accumulator();
+* // returns 24.5
+*
+* @example
+* var incrnanvariance = require( '@stdlib/stats/incr/nanvariance' );
+*
+* var accumulator = incrnanvariance( 3.14 );
+*
+* var v = accumulator();
+* // returns null
+*
+* v = accumulator( 2.0 );
+* // returns 1.2996
```

Comment:
If these are approximate values, you need to include a tilde. Applies here and elsewhere.


```suggestion
* // returns ~1.2996
```

Reply:
okay !